junit5/chapter.md

```diff
@@ -202,14 +203,16 @@ are involved. <br/>
 Taking the issues mentioned above and our initial stakeholder analysis into account, we created a context view of the
 system, as is visible in Fig. 3.
 
-![Context View](https://cloud.githubusercontent.com/assets/12841723/23343910/a747eaaa-fc73-11e6-93a1-24f27d5e90a5.png)
+![Context View](https://cloud.githubusercontent.com/assets/12841723/25781903/140581ea-3340-11e7-9d7e-64852c82c560.png)
```

Comment:
This is unrelated to this particular change, but I think you should make your git repository self contained. Thus, include an explicit `img-junit` folder or something similar, and put your `.png` files there instead of at `githubusercontent.com`.